README.md

```diff
@@ -7,6 +7,25 @@ version for the lifetime of the product.
 
 This repository is solely maintained by Docker, Inc.
 
+:warning: **This repository is now deprecated and will be archived** :warning:
+
+Starting with the Docker 20.10 release, packages for the Docker Engine and
+Docker CLI are built directly from their respective source repositories instead
+of from this repository.
+
+Practically this means:
+1. This repository is no longer the “source of truth” for Docker CE builds.
+1. The commit SHA and tag for Docker CLI build will come from the
```

Comment:
I tend to still number each (but know it's not needed for the rendered Markdown). No need to change if there's no other review comments though

Reply:
Happy to change that :)